alfredo-zimperz/precourse/console-calculator/calculator.js

```diff
@@ -0,0 +1,147 @@
+// ------ ***** Main ***** ------
+
```

Comment:
lo que comentamos ayer de los espacios tras for, function, if, else, etc.... y recuerda primero constantes y despues variables

Reply:
Corregido